doc/PLUGINS.md

```diff
@@ -162,6 +163,10 @@ The `dynamic` indicates if the plugin can be managed after `lightningd`
 has been started using the [plugin][lightning-plugin] JSON-RPC command. Critical plugins that should not be stopped should set it
 to false. Plugin `options` can be passed to dynamic plugins as argument to the `plugin` command .
 
+The `fields_filtering` field indicates if the plugin's rpc calls can
+handle the `_filter` parameter.  If your plugin supplies rpcmthods,
+you should implement this!
+
```

Comment:
An alternative to this would have been to slightly break with the JSON-RPC spec and allow us to add meta-parameters in the top-level dict, instead of in the `payload` keyed dict. This is would have been nice since the filtering, and eventually paging, is orthogonal to the actual operations we're asking `lightningd` to perform.

That would then also have meant that plugins that do not support such post-processing of the results are simply oblivious to it, and we don't need a magic prefix to differentiate method arguments from result post-processing instructions.

Reply:
OK, I have done this instead!